### ISSUE TITLE
[Bug]: Issues loading fish images

### ISSUE THREAD

**Issue by sbreitbart-NOAA** [nmfs-ost/asar#62]
### Describe the bug

The fish cover image won't load using absolute filepaths if using the add_image = TRUE and spp_image arguments. The image will only load once it's been placed in the directory with the new skeleton .qmd file, and spp_image calls the filename only- no filepath (e.g., "Rex_sole.png").

### Describe how to reproduce the bug

# This does work:
```
asar::create_template(
        new_template = TRUE,
        format = "pdf",
        office = "NWFSC",
        species = "Rex sole",
        spp_latin = "Glyptocephalus zachirus",
        year = 2010,
        add_image = TRUE, 
        spp_image = "Rex_sole.png"
    )
```

# This doesn't work:
```
asar::create_template(
        new_template = TRUE,
        format = "pdf",
        office = "NWFSC",
        species = "Rex sole",
        spp_latin = "Glyptocephalus zachirus",
        year = 2010,
        add_image = TRUE, 
        spp_image = "C:/Users/sophie.breitbart/Documents/R_projects/asar/inst/resources/spp_img/Rex_sole.png"
    )
```

### Which OS are you seeing the problem on?

Windows

### Which version of asar are you seeing the problem on?

0.0.0.9000

**Comment by Schiano-NOAA** [comment]
This is failing because the addition of a species image should be just the image. For example "species_images.png". The user should move the image to the support_files folder where the current assessment is being rendered. I think it will be better to add the image to the folder rather than call it so then they can keep everything together and it will be there if they use asar in the future to make the next assessment.

**Comment by sbreitbart-NOAA** [comment]
The support_files folder is only created after entering `create_template()`, right? It might be confusing for the user to have to use that function, add the image, and then potentially enter `create_template()` again so that R can find the image. Is that the proposed workflow, or am I missing something?

**Comment by Schiano-NOAA** [comment]
Resolved with PR #104
